optuna/storages/_rdb/alembic/env.py

```diff
@@ -43,7 +43,9 @@ def run_migrations_offline():
 
     """
     url = config.get_main_option("sqlalchemy.url")
-    context.configure(url=url, target_metadata=target_metadata, literal_binds=True)
+    context.configure(
+        url=url, target_metadata=target_metadata, literal_binds=True, render_as_batch=True
```

Comment:
I added `render_as_batch=True` to allow `ALTER TABLE` with sqlite. See [here](https://stackoverflow.com/questions/30378233/sqlite-lack-of-alter-support-alembic-migration-failing-because-of-this-solutio).

Reply:
[Comment] We may be able to omit `render_as_batch=True` because we use [`Operations.batch_alter_table`](https://alembic.sqlalchemy.org/en/latest/ops.html#alembic.operations.Operations.batch_alter_table). But, I don't think this option causes side-effect and we can keep it as it is.